CHANGELOG.md

```diff
@@ -22,6 +22,39 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 #### Compatible Versions
 
 
+## [7.2.2] - 2024-03-13
+
+### Overview
+
+Bugfix
+
+### EDC UI
+
+https://github.com/sovity/edc-ui/releases/tag/v2.4.0
+
+### EDC Extensions
+
+#### Major Changes
+
+#### Minor Changes
+
+#### Patch Changes
+
+- DspCatalogService: Stable Contract Offer IDs removed
+
+### Deployment Migration Notes
+
+_No special deployment migration steps required_
+
+#### Compatible Versions
+
+- Connector Backend Docker Images:
+    - Dev EDC: `ghcr.io/sovity/edc-dev:7.2.2`
+    - sovity EDC CE: `ghcr.io/sovity/edc-ce:7.2.2`
+    - MDS EDC CE: `ghcr.io/sovity/edc-ce-mds:7.2.2`
+- Connector UI Docker Image: `ghcr.io/sovity/edc-ui:2.4.0`
```

Comment:
Can we stay on edc-ui 2.4.0 if we merge it into main and do a release from main or do we need to upgrade to the latest v2.5.0? https://github.com/sovity/edc-ui/releases/tag/v2.5.0

Reply:
No need to migrate, as 2.5.0 was only released for the broker